grid/__init__.py

```diff
@@ -41,7 +42,7 @@ def check_dependency(
 ):
     if verbose:
         sys.stdout.write("\tChecking for " + str(lib) + " dependency...")
-    o = gr_utils.exec_os_cmd(lib)
+    o = gr_utils.execute_command(lib)
```

Comment:
Oh, I meant a better name to the `o` variable. Can you rename it to something like: `commands`. I should have been more explicit sorry.